services/galley/schema/src/V82_CreateTableTeamAdmin.hs

```diff
@@ -0,0 +1,36 @@
+-- This file is part of the Wire Server implementation.
+--
+-- Copyright (C) 2023 Wire Swiss GmbH <opensource@wire.com>
+--
+-- This program is free software: you can redistribute it and/or modify it under
+-- the terms of the GNU Affero General Public License as published by the Free
+-- Software Foundation, either version 3 of the License, or (at your option) any
+-- later version.
+--
+-- This program is distributed in the hope that it will be useful, but WITHOUT
+-- ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+-- FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more
+-- details.
+--
+-- You should have received a copy of the GNU Affero General Public License along
+-- with this program. If not, see <https://www.gnu.org/licenses/>.
+
+module V82_CreateTableTeamAdmin
+  ( migration,
+  )
+where
+
+import Cassandra.Schema
+import Imports
+import Text.RawString.QQ
+
+migration :: Migration
+migration = Migration 82 "Create table `team_admin`" $ do
+  schema'
+    [r|
+        CREATE TABLE team_admin (
+            team   uuid,
+            user   uuid,
+            PRIMARY KEY (team, user)
+        );
```

Comment:
Please use LeveledCompactionStrategy, it's not cassandra's default, but for almost all kind of data we have at Wire it's the better choice.

```suggestion
        ) with compaction = {'class': 'LeveledCompactionStrategy'};
```

Reply:
Thanks! Updated schema https://github.com/wireapp/wire-server/pull/3431/commits/05ff0045c429763f797e46ebbe101dded772705f